mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/JPAId.java

```diff
@@ -25,7 +25,7 @@
     public static class Factory implements MailboxId.Factory {
         @Override
         public JPAId fromString(String serialized) {
-            return of(Long.parseLong(serialized));
+            return of(Long.valueOf(serialized));
```

Comment:
Maybe. History is not super clean on this PR. Should we invest time cleaning it ?

(And to answer specifically, it is an answer to a comment made by @mbaechler on #460, which might have been stash with something else)